src/pages/resultsView/SurvivalStoreHelper.ts

```diff
@@ -50,3 +51,46 @@ export function getPatientSurvivals(
         return [];
     }
 }
+
+export function getPatientSurvivalUniqueData(
+    survivalClinicalDataGroupByUniquePatientKey: any,
```

Comment:
seems like this is survivalClinicalDataGroupByUniquePatientKey:ClinicalData[]. ?

Reply:
or sorry, [key:string] : ClinicalData[]

Reply:
i think if you just cast this input property, then all the casting inside will be unnecessary.